### DISCUSSION TITLE
how do you play this

### DISCUSSION THREAD

**Discussion by makargh0st** [Jcw87/c2-sans-fight#114] [Q&A]
how do you play this

**Comment by thelift112** [comment]
Please explain in what way you mean, if it's just how to move, use the "ARROW KEYS" or "WASD" to move, "Z" or "ENTER" to select, and "X" or "BACKSPACE" to go back if that is what you're wondering. And if that's what you're wondering you can mark this as the answer.

**Reply by makargh0st** [reply]
the website is blocked is there any other way

**Reply by Jcw87** [reply]
If you can post comments here, you can download files from the repository. The `gh-pages` branch has everything you need to run it. Plop it on to any HTTP server and it will work. If you do this, I request that this HTTP server is not accessible from the internet. Theoretically, it should also work by just opening the `index.html` file, but in practice, some browsers choose to disable some features when it is opened this way.

**Comment by Jcw87** [comment]
He's probably having difficulty finding the link on the main page to the hosted game. It's under the "About" section on the right side of the page. http://jcw87.github.io/c2-sans-fight/
